### DISCUSSION TITLE
Improve "Load more" functionality for long PRs

### DISCUSSION THREAD

**Discussion by IvanPizhenko** [community/community#17446] [Pull Requests]
I have a PR with **lots** of comments and commits... And so I get something like this:
![image](https://user-images.githubusercontent.com/11859904/170841499-3131e88c-f358-4675-875c-dcafcd155ce3.png)
Each time I click it it loads some more stuff, but.... from the beginning, i.e. if there are N items in total and 1 to M were displayed so far, when I click "load more", it show me 1 to M+K items. 

**Improvement 1.** I wish it could have functionality where there would be 2 links (below and above "X hidden items"), and when I click the upper one, it works as now, but when I click the bottom link, it would show items 1... M, then this "X hidden items", and then items N - K to N. 

**Improvement 2.** It also would be good to have there link called "Load all", so I can load all at once if I wish so instead of multiple times clicking "Load more", waiting time after each such click...

Please consider implementing  at least first one. Thanks in advance...

**Comment by pierre-eliep-met** [comment]
Agreed, handling long PR is sometimes painful because of this, it is not easy to find the comments we're working on

But overall I think this feedback shows that currently long PRs are not well handled with this very linear reviewing process

**Reply by garry-ut99** [reply]
Duplicate of: ["Load more comments" in issues and PRs needs to work better](https://github.com/orgs/community/discussions/5119)
